pages/web/domains/dns_zone_spf/guide.es-es.md

```diff
@@ -162,7 +148,11 @@ Una vez completada la información, haga clic en `Siguiente`{.action}, asegúres
 
 #### Utilizar el registro SPF de OVHcloud <a name="spfrecordovhcloud"></a>
 
-Ha elegido el registro `SPF`{.action} y quiere aplicar la configuración de OVHcloud.
+Ha elegido el registro `SPF`{.action} y quiere aplicar la configuración de OVHcloud. que permite incluir todos los servidores de correo salientes de OVHcloud para los siguientes servicios de correo:
```

Comment:
Please fix the 2nd sentence, it should start differently (not with "que permite").